resolwe_bio/tests/processes/test_upload.py

```diff
@@ -180,7 +180,8 @@ def test_upload_paired_end_reads(self):
         wrong_mates = self.run_process("upload-fastq-paired", inputs, Data.STATUS_ERROR)
         error_msg = [
             "The number of mate-pair files in split-lane samples must match. 2 and 1 "
-            "input files were given for the -fq and -fq2 inputs, respectively."
+            "input files were given for the -fq and -fq2 inputs, respectively.",
```

Comment:
I think there should be a space before string closes.

Reply:
There are two elements in the error msg list object. It's not one string.